testsuite/tests/tool-ocamldep-strict/strict_modules_error.ml

```diff
@@ -0,0 +1,8 @@
+(* TEST
+modules = "dep.ml a.ml"
```

Comment:
I think it is not necessary to test both with, and without `-modules`, since the behaviour being changed is orthogonal to this flag. I would leave only the one without `-modules` and remove the one with `-modules`.

Reply:
I removed the ones with -modules. I'd like to implement @shindere 's suggestion to group things in a single test, but I'm not sure to do it. This is my first time with ocamltest. How can I specify that the same test should be run with different options and different expected outcomes?